### ISSUE TITLE
Hierarchy header with EditorOnly tag by default

### ISSUE THREAD

**Issue by Ardoos** [arimger/Unity-Editor-Toolbox#4]
Reason: Hierarchy header created with right click Editor Toolbox -> Hierarchy Header has EditorOnly tag by default.

My story: I started to use your extension and one of the helpful with was to setup hierarchy with custom hierarchy headers. At the beginning I was creating them with right click Editor Toolbox -> Hierarchy Header. Everything worked fine in the editor and then I build my game on Android device. I hit start and the only thing I have is blue background from my camera. After 2h of debugging my code, no errors in logcat, I found out that most of my objects are child of mentioned headers that have **EditorOnly** tag.

Question: I am not quite sure what's the point of doing headers with EditorOnly tag by default (and not pointing it out with bolded font in the feature description). Of course you can just use #h prefix on empty gameobject, but the second option shouldn't be a trap imho.
I hope I can learn why this header is used with EditorOnly tag by default, maybe I misunderstood something and I'm not using it in the correct way. Anyway, maybe I just saved 2 hourse of someone's development time :)

**Comment by arimger** [comment]
Hello there!

So basically when I added Hierarchy Headers I treated them like editor-only addition, that's why in my all projects they are objects without parents, children, and components, used only to display dedicated information. The main reason for using **EditorOnly** tag is to keep the unnecessary objects out of builds. 

I'm really sorry you lost 2h in this case, I completely forgot to mention this information in the README.md file.
